lib/analytics/course_students_csv_builder.rb

```diff
@@ -17,6 +22,28 @@ def generate_csv
 
   private
 
+  def populate_created_articles
+    # A user has created an article during the course if
+    # the user is in the user_ids of new_articles_courses
+    # has a revision with new_article: true for that article
+    @course.new_articles_courses.pluck(:article_id, :user_ids).each do |article_id, user_ids|
+      user_ids.each do |user_id|
+        @created_articles[user_id] += 1 if article_creator?(article_id, user_id)
+      end
+    end
+  end
```

Comment:
This goes through the results of new articles and checks which user in the user_ids is the creator by looking at the revisions of the course.
at the end of this method the created_articles should be a hash with user_id as keys and the count of created articles as values.